docs/content/concepts/metadata-tags/asset-metadata/table-metadata.mdx

```diff
@@ -0,0 +1,192 @@
+---
+title: "Table metadata | Dagster Docs"
+description: "Table metadata can be used to provide additional context about a tabular asset, such as its schema, row count, and more."
+---
+
+# Table metadata
+
+Table metadata provides additional context about a tabular asset, such as its schema, row count, and more. This metadata can be used to improve collaboration, debugging, and data quality in your data platform.
+
+Dagster supports attaching different types of table metadata to assets, including:
+
+- [**Column schema**](#attaching-column-schema): Describes the structure of the table, including column names and types
+- [**Row count**](#attaching-row-count): Describes the number of rows in a materialized table
+- [**Column-level lineage**](#attaching-column-level-lineage): Describes how a column is created and used by other assets
+
+---
+
+## Attaching column schema
+
+### For assets defined in Dagster
+
+<Image
```

Comment:
Can we add a sentence here that describes what this image shows?

Reply:
Hmm, I'm finding it hard to come up with a way to describe it that doesn't interrupt the intro flow of this section. I included the image to frontload what the user can expect to get out of that section - it could also go at the bottom as an example of what the code outputs, but I do think having it at the top is nice in that it prompts users to read more if they are enticed by the image.

Reply:
Thoughts on approaching it like this?

> Column schema metadata can be attached to Dagster assets either as [definition metadata](/concepts/metadata-tags/asset-metadata#attaching-definition-metadata) or [materialization metadata](/concepts/metadata-tags/asset-metadata#attaching-materialization-metadata), which will then be visible in the Dagster UI. For example:
>
> [image]
>
> If the schema of your asset is pre-defined...

Reply:
Good idea - updated!